readthedocs/settings/base.py

```diff
@@ -181,6 +181,24 @@ def USE_PROMOS(self):  # noqa
         'allauth.account.auth_backends.AuthenticationBackend',
     )
 
+    AUTH_PASSWORD_VALIDATORS = [
+        {
+            'NAME': 'django.contrib.auth.password_validation.UserAttributeSimilarityValidator',
+        },
+        {
+            'NAME': 'django.contrib.auth.password_validation.MinimumLengthValidator',
```

Comment:
I really dislike these defaults in dev. I think these should likely only be prod settings.